### DISCUSSION TITLE
how to limit the popup result

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#16975] [How to]
I created a contact field in quotes,  which is to select contact from contacts record,  when I choose contact, I want the pop up record to be limited by the account which I already chose. how to make it?

![1013050219](https://user-images.githubusercontent.com/18006681/226809510-0cbbe1ad-ee1d-4538-96f0-77e6af6a90e2.png)
![1649691805](https://user-images.githubusercontent.com/18006681/226809528-455d14ab-5cf9-46ee-b0a9-54d6086e880b.png)

**Comment by newminder** [comment]
like this, but need me to type the name there in member of

![26766503](https://user-images.githubusercontent.com/18006681/226810550-0a81d88a-ff02-4fe9-be0a-94a5c6f431fa.png)
![2096234478](https://user-images.githubusercontent.com/18006681/226810614-0c18c184-5d9e-4270-8269-b76dce24d9ea.png)

**Comment by mariuszkrzaczkowski** [comment]
Here is the code responsible for it, do the same as in the example below
https://github.com/YetiForceCompany/YetiForceCRM/blob/8b3d0cb0cc12890e05c59f199129e8adbeaab5eb/app_data/moduleHierarchy.php#L58-L61

**Reply by newminder** [reply]
this does not work. i tried.  still show all contacts, not the ones just related to the account

**Reply by newminder** [reply]
> Here is the code responsible for it, do the same as in the example below
> 
> https://github.com/YetiForceCompany/YetiForceCRM/blob/8b3d0cb0cc12890e05c59f199129e8adbeaab5eb/app_data/moduleHierarchy.php#L58-L61

my codes are but does not work,  could you please tell me how to write the correct codes to limit popup search result?
'SQuotes' => [
	'accountid' => ['Accounts' => ['company' => ['accountname']]], 
	'salesprocessid' => ['SSalesProcesses' => ['accountid' => ['related_to']]],
			
	'accountid' => [
		 'Contacts' => [
		            'accountid' => ['parent_id'], 
		            'contact'=>['contact_id']
		  ]
	 ], 
],



also  I tried another way in
modules/Contacts/models/Module.php
and add sql codes,  but do not know hot to get the input value of 'accountid'   to the vtiget_contactdetails.parentid, if you know hot to make it, pls tell me too.

**Comment by newminder** [comment]
anyone knows how to make it?  please help.
